src/Builder/FormContractor.php

```diff
@@ -113,16 +119,13 @@ public function getDefaultOptions($type, FieldDescriptionInterface $fieldDescrip
         $options['sonata_field_description'] = $fieldDescription;
 
         switch ($type) {
-            case 'Sonata\DoctrinePHPCRAdminBundle\Form\Type\TreeModelType':
-            case 'doctrine_phpcr_odm_tree':
+            case TreeModelType::class:
                 $options['class'] = $fieldDescription->getTargetEntity();
                 $options['model_manager'] = $fieldDescription->getAdmin()->getModelManager();
 
                 break;
-            case 'Sonata\AdminBundle\Form\Type\ModelType':
-            case 'sonata_type_model':
```

Comment:
Not sure if we should handle this as a BC break. Using form aliases is deprecated since symfony <= 3, but we are about to remove functionality.